extensions/amp-viewer-assistance/validator-viewer-assistance.protoascii

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
This file already exists as [validator-amp-viewer-assistance.protoascii](https://github.com/ampproject/amphtml/blob/master/extensions/amp-viewer-assistance/validator-amp-viewer-assistance.protoascii)

Reply:
Removed.